spec/controllers/api/v1/skills_controller_spec.rb

```diff
@@ -38,32 +43,60 @@
       expect(json_response_skill_group_id).to eq(group.id)
     end
 
-    it "responds with error messages if a skill fails to create" do
-      post :create, format: :json, skill: { name: "" }
-
-      expect(response.status).to eq 422
-      expect(json_response_error_message).to eq(["Name can't be blank"])
+    context "when skill is invalid" do
+      it "returns a 422 status code" do
+        post :create, format: :json, skill: {name: nil}
+        expect(response.status).to eq 422
+      end
+
+      it "responds with error messages if name missing" do
+        post :create, format: :json, skill: {
+                                              name: "",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.